pkg/chunked/storage_linux.go

```diff
@@ -873,6 +923,9 @@ func (c *chunkedDiffer) ApplyDiff(dest string, options *archive.TarOptions) (gra
 	// modifies the source file as well.
 	useHardLinks := parseBooleanPullOption(&storeOpts, "use_hard_links", false)
 
+	// List of OSTree repositories to use for deduplication
+	ostreeRepos := strings.Split(storeOpts.PullOptions["ostree_repos"], ":")
```

Comment:
@giuseppe I might be wrong should the convention be `,` instead of `:` we are using `:` at various places for extra mount args

Reply:
Please ignore if multiple repos are generally seperated with `:` in `storage.conf`

Reply:
`:` is often used as a path separator on unix, e.g. for `PATH`, and golang defines `os.PathListSeparator` as well as `:`.

I am fine to change it though if it is what we use in other places

Reply:
@giuseppe thanks for clarifying i think `:` is better its a generally accepted convention